apps/fluent-tester/src/FluentTester/theme/applyBrand.ts

```diff
@@ -156,10 +174,10 @@ export const applyBrand = (currentBrand: string): PartialTheme => {
           compoundBrandForeground1Pressed: ramp.AppShade20,
           brandForeground1: ramp.AppPrimary,
           brandForeground2: ramp.AppShade10,
-          brandBackground: ramp.AppPrimary,
-          brandBackgroundHover: ramp.AppShade10,
-          brandBackgroundPressed: ramp.AppShade30,
-          brandBackgroundSelected: ramp.AppShade20,
+          brandBackground1: ramp.AppPrimary,
```

Comment:
I'm a bit confused by this naming convention... Why add the "1" suffix?

Reply:
The names for some of the entries are taken from the new alias tokens that design has decided on, some of which have numbers. The other issue here is that brandBackground already exists in a different subtype that makes up the Palette type, so in order to avoid the name clash I decided to append 1 to the brandBackground we're getting from the alias tokens. Since other alias tokens already have that naming convention, I figured this would be the best way to resolve the name conflict.

Reply:
What's the other subtype where brandBackground exists? Is that the ControlColorTokens subtype? I think that may only be used by the experimental button and not by and clients if so.

Reply:
Yeah it's the ControlColorTokens subtype. I can go change that instead if preferred.

Reply:
Does experimental button still render fine with the default theme? I think that's the only control that should be affected

Reply:
@Saadnajmi So I'm looking at the code and I'm not entirely sure how the Experimental Button has worked in the default theme up til now. It has some coloring in master right now but I think that's an incidental thing from the name clash I'm trying to fix in this PR... Works fine in the Office theme though.

Guess it's time to patch the default theme.

Reply:
Though I would want to do that in its own PR if that's alright

Reply:
Oh yeah totally! I didn't mean that to be a blocking comment sorry